tests/test_tui_publish_workflow_security.py

```
@@ -1437,5 +1616,713 @@ def test_stable_release_builds_and_tests_once_before_dispatching_publishers() ->
+            env=environment,
+            check=False,
+            text=True,
+            capture_output=True,
+        )
+        assert result.returncode != 0
+        assert "multiple matching" in result.stderr
+
+
+def test_release_cut_does_not_mask_waiter_failure() -> None:
+    release_cut = workflow("cmux-tui-release-cut.yml")
+    document = yaml.load(release_cut, Loader=yaml.BaseLoader)
```

Ruff (0.16.1)

[error] Probable use of unsafe loader `BaseLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)